core/templates/dev/head/css/oppia.css

```diff
@@ -1307,7 +1307,7 @@ md-card.oppia-dashboard-tile {
   cursor: pointer;
 }
 .toast-warning {
-  background-image: url("/images/general/warning.png");
+  background-image: url("/assets/common/images/general/warning.png");
```

Comment:
It can, but it's in css and we can't dynamically add stuff here. So, we moved it to a separate folder called common.